### PR TITLE
Migrating virt-components article (jsc#DOCTEAM-1280)

### PR DESCRIPTION
**Pull request by tbazant**
### Are there any relevant issues/feature requests?

* bsc#DOCTEAM-1280

### Is this (based on) existing content?

* DC-task-install-virtualization-components
* https://susedoc.github.io/doc-modular/main/html/task-install-virtualization-components/

### CODE REVIEW

**Review by lvicoun — APPROVED** [review]

File: references/virt-components-installation-patterns.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    <meta name="maintainer" content="tbazant@suse.com" its:translate="no"/>
+    <abstract>
+      <xi:include href="../snippets/virt-components-intro.xml"/>
+    </abstract>
+  </info>
+  <para>
+    Related software packages from &productname; software repositories are
+    organized into <emphasis>installation patterns</emphasis>. You can use these
+    patterns to install specific virtualization components on an already running
+    &productname;. Use <command>zypper</command> to install the patterns, for
```
Comment (dariavladykina) [inline references/virt-components-installation-patterns.xml:31]:
```suggestion
    &productname; system. Use <command>zypper</command> to install the patterns, for
```

File: references/virt-components-installation-patterns.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    example:
+  </para>
+<screen>&prompt.sudo;zypper install -t pattern <replaceable>kvm_server</replaceable></screen>
+  <variablelist>
+    <varlistentry>
+      <term><literal>kvm_server</literal></term>
+      <listitem>
+        <para>
+          Installs basic &vmhost; with the &kvm; and &qemu; environments.
```
Comment (dariavladykina) [inline references/virt-components-installation-patterns.xml:40]:
```suggestion
          Installs a basic &vmhost; with the &kvm; and &qemu; environments.
```

File: references/virt-components-installation-patterns.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        </para>
+      </listitem>
+    </varlistentry>
+    <varlistentry>
+      <term><literal>kvm_tools</literal></term>
+      <listitem>
+        <para>
+          Installs &libvirt; tools for managing and monitoring &vmguest;s in
+          &kvm; environment.
```
Comment (dariavladykina) [inline references/virt-components-installation-patterns.xml:49]:
```suggestion
          the &kvm; environment.
```

File: references/virt-components-installation-patterns.xml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+          Installs a basic &xen; &vmhost;.
+        </para>
+      </listitem>
+    </varlistentry>
+    <varlistentry>
+      <term><literal>xen_tools</literal></term>
+      <listitem>
+        <para>
+          Installs &libvirt; tools for managing and monitoring &vmguest;s in
+          &xen; environment.
```
Comment (dariavladykina) [inline references/virt-components-installation-patterns.xml:66]:
```suggestion
          the &xen; environment.
```

File: tasks/virt-components-installation-yast.xml [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  <info>
+    <title>Installing virtualization components using &yast;</title>
+    <meta name="maintainer" content="tbazant@suse.com" its:translate="no"/>
+    <abstract>
+      <xi:include href="../snippets/virt-components-intro.xml"/>
+    </abstract>
+  </info>
+  <para>
+    This topic describes how to install virtualization components on an already
+    installed &productnameshort; using &yast;.
```
Comment (dariavladykina) [inline tasks/virt-components-installation-yast.xml:29]:
```suggestion
    installed &productnameshort; system using &yast;.
```

File: tasks/virt-components-installation-yast.xml [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+        Start &yast; and select <menuchoice><guimenu>Virtualization</guimenu>
+        <guimenu>Install Hypervisor and Tools</guimenu></menuchoice>.
+      </para>
+    </step>
+    <step>
+      <substeps>
+        <step>
+          <para>
+            Select <guimenu>&kvm; server</guimenu> for a minimal installation of
+            &qemu; and &kvm; environment. To use the &libvirt;-based management
```
Comment (dariavladykina) [inline tasks/virt-components-installation-yast.xml:50]:
```suggestion
            the &qemu; and &kvm; environment. To use the &libvirt;-based management
```

File: tasks/virt-components-installation-yast.xml [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+            stack with &kvm; as well, select <guimenu>&kvm; tools</guimenu>.
+          </para>
+        </step>
+        <step>
+          <para>
+            Select <guimenu>&xen; server</guimenu> for a minimal installation of
+            &xen; environment. To use the &libvirt;-based management stack with
```
Comment (dariavladykina) [inline tasks/virt-components-installation-yast.xml:57]:
```suggestion
            the &xen; environment. To use the &libvirt;-based management stack with
```

File: articles/virt-components.asm.xml [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+<!-- For a full list of meta tags and their values,
+	   see https://confluence.suse.com/x/aQDWNg
+      -->
+      <meta name="architecture" its:translate="no"><phrase>&x86-64;</phrase><phrase>&power;</phrase>
+      </meta>
+      <meta name="productname" its:translate="no">
+<!-- enter product name and version --><productname version="X.Y">&productname;</productname>
+      </meta>
+      <meta name="title" its:translate="yes">Server virtualization components</meta>
+      <meta name="description" its:translate="yes">Server virtualization components</meta>
```
Comment (dariavladykina) [inline articles/virt-components.asm.xml:71]: This seems too short for a meta description. How about: _Installing virtualization components on the server to run a virtualization server that can host multiple guest systems_?

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions here. Thanks!

File: articles/virt-components.asm.xml (hunk 8 above)
Comment (tbazant) [inline articles/virt-components.asm.xml:71]:
```suggestion
      <meta name="description" its:translate="yes">Installing virtualization components on a virtualization server that can host multiple guest systems</meta>
```

**Review by tbazant — COMMENTED** [review]
one small suggestion

**Pull request merged**